### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Victor-Mirandas**
- [DEMO LINK](https://Victor-Mirandas.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Victor-Mirandas.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,46 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#" class="img"><img src="/src/images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul>
+          <li>
```
Comment (Viktor-Kost) [inline src/index.html:18]: add seelctors
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Comment by Victor-Mirandas** [comment]
[DEMO LINK](https://victor-mirandas.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://victor-mirandas.github.io/layout_moyo-header/report/html_report/)

**Comment by Victor-Mirandas** [comment]
[DEMO LINK](https://victor-mirandas.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://victor-mirandas.github.io/layout_moyo-header/report/html_report/)

**Comment by Victor-Mirandas** [comment]
[DEMO LINK](https://victor-mirandas.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://victor-mirandas.github.io/layout_moyo-header/report/html_report/)

**Review by Victor-Mirandas — COMMENTED** [review]
- [DEMO LINK](https://Victor-Mirandas.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Victor-Mirandas.github.io/layout_moyo-header/report/html_report/)

**Comment by Victor-Mirandas** [comment]
- [DEMO LINK](https://Victor-Mirandas.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Victor-Mirandas.github.io/layout_moyo-header/report/html_report/)

**Pull request closed without merging**